playground-common/src/Schema.hs

```diff
@@ -88,6 +89,7 @@ data FormSchema
     -- Blessed types that get their own special UI widget.
     | FormSchemaValue
     | FormSchemaSlotRange
```

Comment:
kill it?

Reply:
Shouldn't be kept since we should keep users on the Playground to use slots?